stdlib/universe/table_fns.go

```diff
@@ -299,3 +314,139 @@ func objectFromRow(idx int, cr flux.ColReader) values.Object {
 	}
 	return values.NewObjectWithValues(vsMap)
 }
+
+func NewFindColumnFunction() values.Value {
+	return values.NewFunction("findColumn",
+		runtime.MustLookupBuiltinType("universe", "findColumn"),
+		findColumnCall,
+		false)
+}
+
+func findColumnCall(ctx context.Context, args values.Object) (values.Value, error) {
+	arguments := interpreter.NewArguments(args)
+	v, err := arguments.GetRequired(tableFindStreamArg)
+	if err != nil {
+		return nil, err
+	}
+	to, ok := v.(*flux.TableObject)
+	if !ok {
+		return nil, errors.Newf(codes.Invalid, "expected TableObject but instead got %T", v)
+	}
+
+	var fn *execute.TablePredicateFn
```

Comment:
I noticed that this was passed in as an argument in `tableFind()` but here it is assigned. Just wondering if this changes anything with the way that the table is processed in this call.

Reply:
There is some code duplication there, which isn't ideal. All three of `tableFindCall`, `findColumnCall`, and `findRecordCall` extract the `fn` arg before passing it to the common worker `tableFind`. I'm not sure if we have a standard for dealing with that. Normally when I'm just pulling args from an environment I will duplicate code and make any workers common shared code, but I'm not sure how others feel about that.